pkg/platform/packet/packet_test.go

```diff
@@ -266,6 +291,7 @@ func baseConfig() *config {
 				Name: "2",
 			},
 		},
+		NodePrivateCIDR: "11.11.11.11",
```

Comment:
I don't think this should be part of the base config, as this field is deprecated.